src/components/PersonLink.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const params = useParams();
+  const { sex, name, born, died, fatherName, motherName, slug } = person;
+
+  const mother = motherName ? people.find(p => p.name === motherName) : null;
+
+  const father = fatherName ? people.find(p => p.name === fatherName) : null;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': params.slug === slug })}
+    >
+      <td>
+        <Link
+          className={cn({ 'has-text-danger': sex === 'f' })}
+          to={`/people/${slug}`}
+        >
+          {name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {mother ? (
+          <Link className="has-text-danger" to={`/people/${mother.slug}`}>
+            {mother.name}
+          </Link>
+        ) : motherName ? (
+          motherName
+        ) : (
+          '-'
```

Comment:
You've handled the case where a mother is not found in the `people` array by displaying the `motherName`. However, you should consider what should happen if `motherName` is also `null` or an empty string. The current code would display an empty string instead of the placeholder '-'.